pkg/eventcounter/api/api.go

```diff
@@ -101,6 +106,86 @@ func (s *eventCounterService) Register(server *grpc.Server) {
 	ecproto.RegisterEventCounterServiceServer(server, s)
 }
 
+// TODO temporary name
+func (s *eventCounterService) GetEvaluationCountBigQuery(
```

Comment:
No need to change it now. I'm just sharing some ideas for the API name.

- GetExperimentEvaluationCount
- GetExperimentGoalCount